Build fails on Windows:

```
Error: D:\a\cpython\cpython\Modules\_sqlite\statement.c(117,27): error C2198: '_pysqlite_seterror': too few arguments for call [D:\a\cpython\cpython\PCbuild\_sqlite3.vcxproj]
```

--- a/Modules/_sqlite/statement.c
+++ b/Modules/_sqlite/statement.c
@@ -114,7 +114,7 @@ pysqlite_statement_create(pysqlite_Connection *connection, PyObject *sql)
     PyObject_GC_Track(self);
 
     if (rc != SQLITE_OK) {
-        _pysqlite_seterror(self->db);
+        _pysqlite_seterror(self->db, NULL);
         goto error;
     }
 

